apps/examples/wc-next10/pages/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Head from 'next/head';
+import dynamic from 'next/dynamic'
+
+const PButton = dynamic(() => import('../ButtonLogic'), { ssr: false });
+
+export default function Home() {
+    return (
+        <>
+            <Head>
+                <title>Create Next App</title>
+                <meta name="viewport" content="width=device-width, initial-scale=1" />
+                <link rel="icon" href="/favicon.ico" />
+            </Head>
+            <main>
+                <PButton />
```

Comment:
Here's what I'd suggest to clean this up (I'm sorry it's gone on over a few comments):

Rename `ButtonLogic.js` to `Counter.js`. Rename the exported component from `PButton` to `Counter`. 

I am happy to hop on a call and explain my reasoning a little more if you'd like! But in a nut shell, even though this is an example app I think we should try to keep the architecture clean and reasonably close to what we'd expect from an actual NextJS application.

_Caveat: I am not super experienced with Next/React so perhaps @raoufswe could weigh in and validate/disagree with my suggestion? 😂 _

Then in here just use `<Counter />`

Reply:
I agree, I also think renaming it to `Counter` encapsulates what the component actually does, not, what element it uses.